.travis.yml

```diff
@@ -39,65 +39,118 @@ install:
   - pip install flake8-future-import parameterized sphinx_rtd_theme
   # nose-exclude plugin allow us to tell nosetests to exclude folder with --exclude-dir=path/to/directory.
   - pip install nose-exclude nose-timer
-  - conda install --yes -q scipy=0.19.1  # Try to reinstall it to fix the problem
+  - if [[ $NUMPY_VERSION == '1.13.0' ]]; then conda install --yes -q scipy=0.19.1; else conda install --yes -q scipy=0.14; fi  # Try to reinstall it to fix the problem
 
 jobs:
   include:
     # define prototype for doctest
     - &doctest
       stage: doc
       python: "2.7"
-      env: DOC=1 PART="theano/tests/test_flake8.py"
+      env: NUMPY_VERSION=1.9.1 DOC=1 PART="theano/tests/test_flake8.py"
     # re-use prototype, changing the Python version
+    - <<: *doctest
+      python: "2.7"
+      env: NUMPY_VERSION=1.13.0 DOC=1 PART="theano/tests/test_flake8.py"
     - <<: *doctest
       python: "3.4"
+      env: NUMPY_VERSION=1.9.1 DOC=1 PART="theano/tests/test_flake8.py"
+    - <<: *doctest
+      python: "3.6"
```

Comment:
I would do 3.7, if we support 3.4 and 3.7 it's fairly unlikely we wouldn't support 3.6 as well.